azurerm/internal/services/datafactory/data_factory_linked_service_azure_databricks_resource.go

```diff
@@ -56,40 +56,22 @@ func resourceDataFactoryLinkedServiceAzureDatabricks() *schema.Resource {
 			"resource_group_name": azure.SchemaResourceGroupNameDiffSuppress(),
 
 			// Authentication types
-			"authentication_msi": {
-				Type:     schema.TypeList,
-				MaxItems: 1,
-				Optional: true,
-				Elem: &schema.Resource{
-					Schema: map[string]*schema.Schema{
-						"workspace_resource_id": {
```

Comment:
We can shorten this down to just a string since it's just one Item in a one size list and I don't imagine we'll add anything to this list.